docs/quickstart.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: default
+title: Dev Quick Start Guide
+nav_order: 7
+---
+
+# Dev Quick Start Guide
+{: .no_toc }
+
+This document describes how to quickly get a runtime server built with `sample_lib` library, train a model with gRPC and with that trained model, send an inference call to the server with either HTTP or gRPC call.
+
+## Pre-reqs
+To set up development environment and tooling, the [Development Guide]({{ site.baseurl }}{% link docs/developing.md %}) page is a good place to start.
+
+## Build and start a runtime server with sample_lib
+
+Run the `start_runtime_with_sample_lib` python script:
```

Comment:
Is it worth duplicating the information that we have in the Caikit examples [README](https://github.com/caikit/caikit/blob/main/examples/sample_lib/README.md)? If we want to add more information, I feel we should add it to that readme instead of duplicating it here? That would make this doc pretty small though